.storybook/routes.js

```diff
@@ -0,0 +1,61 @@
+// List of routes The key is the Canvas Site URL The value is the first Storybook example ID. This
+// prevents odd scrolling. This file is used by the `webpack-loader-redirect-mdx-to-github` loader
+// and will rewrite the URL to point to the correct URL for the story. The Storybook key will be
+// rewritten to something like `?path=/docs/${id}`
+const routes = {
+  'getting-started/introduction': 'welcome-getting-started--page',
```

Comment:
This should include the beginning and trailing slashes as well, right?

```suggestion
  '/getting-started/introduction/': 'welcome-getting-started--page',
```

Also, just to clarify, we're including a mapping for every route we think could be linked to from an MDX exported by `@workday/canvas-kit-docs`? That's why `/getting-started/introduction/` is here (because we may want to link to it from, say, `CONTRIBUTING.mdx`), but `/content/overview/` isn't (because it's unlikely we'd want to link to it from anything in `@workday/canvas-kit-docs`).

In both cases, there's no `canvas-kit` equivalent: [`/getting-started/introduction/`](https://ghe.megaleo.com/pages/design/canvas-site/getting-started/introduction) and [`welcome-getting-started--page`](https://workday.github.io/canvas-kit/?path=/docs/welcome-getting-started--page) aren't the same, and [`/content/overview/`](https://ghe.megaleo.com/pages/design/canvas-site/content/overview) isn't remotely close to anything in `canvas-kit`. The difference is `/getting-started/introduction/` may need to be linked to from an MDX in `@workday/canvas-kit-docs` so we need to provide a mapping for it (even if it's an imperfect one).

Am I understanding that right?

Reply:
Yes, I did forget the leading slash. I am trying to make a 1:1 mapping of links. It is possible that the getting started page is really a dev getting started page and should actually be linked to something like `/getting-started/for-developers/introduction/`.

There were many links that I say no equivalent for. It is possible that there's a special value type that allows us to link from Storybook straight to the Canvas Site since the content may only live there. Something like `canvas-site` or something that indicates no perfect matching and to redirect to the canvas site instead. This list will be subject to change over time and will be necessary to keep links up-to-date with Canvas Site content.

The main goal  is to keep interlinking mostly working within Storybook and Canvas Site without a lot of weird loops. For example, I've entirely been avoiding linking between documents to avoid the issue.